launchcontainer/tests.py

```diff
@@ -55,7 +55,7 @@ def make_one(self, display_name=None, **kw):
         """
         Creates a launchcontainer XBlock for testing purpose.
         """
-        from launchcontainer import LaunchContainerXBlock as cls
+        from .launchcontainer import LaunchContainerXBlock as cls
```

Comment:
This is weird. I think `launchcontainer` is available without the dot prefix and this is just 2to3 noise. Correct?

```suggestion
        from launchcontainer import LaunchContainerXBlock as cls
```